origin-bridge/src/controllers/redirects.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+const express = require('express')
+const router = express.Router()
+
+router.get('/facebook', (req, res) => {
+  if (req.query.dappRedirectUrl) {
+    const dappRedirectUrl = req.query.dappRedirectUrl
+    res.redirect(dappRedirectUrl + '?origin-code=' + req.query.code)
```

Comment:
ultra nitpick: can be expressed without string concatenation
```
`${dappRedirectUrl}?origin-code=${req.query.code}`
```

Reply:
👍Good spot, I prefer this too